libuavcan_drivers/stm32/driver/src/uc_stm32_can.cpp

```diff
@@ -450,10 +450,35 @@ uavcan::int16_t CanIface::receive(uavcan::CanFrame& out_frame, uavcan::Monotonic
 uavcan::int16_t CanIface::configureFilters(const uavcan::CanFilterConfig* filter_configs,
                                            uavcan::uint16_t num_configs)
 {
-    // TODO: Hardware filter support
-    CriticalSectionLocker lock;
-    (void)filter_configs;
-    (void)num_configs;
+    if (num_configs < (NumFilters / 2)) {
```

Comment:
`NumFilters` is set to 14, which is already half of the amount of filters available on dual-CAN devices. Therefore I suggest to remove the division by two here and below.